torch/onnx/symbolic_opset9.py

```diff
@@ -787,12 +787,14 @@ def _reduce_with_dtype(onnx_op: str, name: str, allow_multi_dim_support: bool =
 
 
 @_onnx_symbolic("aten::cumsum")
-@symbolic_helper.parse_args("v", "i", "none")
+@symbolic_helper.parse_args("v", "i", "none", "b")
 @_beartype.beartype
-def cumsum(g: jit_utils.GraphContext, input, dim, dtype):
+def cumsum(g: jit_utils.GraphContext, input, dim, dtype, full):
```

Comment:
Is it because this one does not have default value as the other one, so it's caught by BC_linter?